<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>372</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Demonio - Cliente</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QFrame" name="frameShellRemota">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>6</y>
      <width>150</width>
      <height>114</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="1" column="0">
      <widget class="QLineEdit" name="entradaTexto"/>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="botonEnviar">
       <property name="text">
        <string>Enviar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QTextEdit" name="salidaTexto"/>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frameConexion">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>720</y>
      <width>570</width>
      <height>227</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QTreeWidget" name="arbolConectados">
         <column>
          <property name="text">
           <string notr="true">Servidores y recursos</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="botonAnadirServidor">
           <property name="text">
            <string>A침adir servidor</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="botonBorrarServidor">
           <property name="text">
            <string>Borrar servidor</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="botonDesconectar">
           <property name="text">
            <string>Desconectar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>   Opciones del servidor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="botonCerrarServidor">
         <property name="text">
          <string>Cerrar servidor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="botonReiniciar">
         <property name="text">
          <string>Reiniciar servidor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="botonDesinfectar">
         <property name="text">
          <string>Desinstalar </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="botonOpcionesXMPP">
         <property name="text">
          <string>Opciones XMPP</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="notificacionLabel">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>317</y>
      <width>85</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Sin notificaciones</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listaOpciones">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>6</y>
      <width>231</width>
      <height>114</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentRow">
     <number>-1</number>
    </property>
    <item>
     <property name="text">
      <string>Conexi칩n</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../icons/camera-web.png</normaloff>
       <normalon>../../../icons/modem.png</normalon>../../../icons/camera-web.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Informaci칩n del sistema</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Administraci칩n del sistema</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Administrador de procesos</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Consola remota</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>../../../icons/shell.png</normalon>
      </iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Administrador de archivos</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>drive-harddisk.png</normaloff>
       <normalon>../../../icons/drive-harddisk.png</normalon>drive-harddisk.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Escritorio remoto</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>user-desktop.png</normaloff>
       <normalon>../../../icons/user-desktop.png</normalon>user-desktop.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Webcam</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>../../../icons/camera-web.png</normalon>
      </iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Keylogger</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>input-keyboard.png</normaloff>
       <normalon>../../../icons/input-keyboard.png</normalon>input-keyboard.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mensajes emergentes</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Diversion</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>face-smile-big.png</normaloff>
       <normalon>../../../icons/face-smile-big.png</normalon>face-smile-big.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Chat</string>
     </property>
    </item>
   </widget>
   <widget class="QFrame" name="frameLicencia">
    <property name="geometry">
     <rect>
      <x>229</x>
      <y>6</y>
      <width>150</width>
      <height>114</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QTextEdit" name="licenciaText"/>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frameArchivos">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>126</y>
      <width>217</width>
      <height>39</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="botonArchivos">
       <property name="text">
        <string>Abrir adminisrador de archivos remoto</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frameMensajes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>348</width>
      <height>239</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QGroupBox" name="tipoMensaje">
       <property name="title">
        <string>Tipo de mensaje</string>
       </property>
       <widget class="QRadioButton" name="radioInformacion">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>84</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Informacion</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioAlerta">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>84</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Alerta</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioPeligro">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>84</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Peligro</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="labelTitulo">
         <property name="text">
          <string>Titulo de la ventana</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textoTitulo"/>
       </item>
       <item>
        <widget class="QLabel" name="labelTexto">
         <property name="text">
          <string>Texto de la ventana</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textoMensaje"/>
       </item>
       <item>
        <widget class="QPushButton" name="botonMostrar">
         <property name="text">
          <string>Mostrar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frameEscritorio">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>310</y>
      <width>153</width>
      <height>39</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QPushButton" name="botonEscritorio">
       <property name="text">
        <string>Mostrar escritorio remoto</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frameWebcam">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>212</width>
      <height>45</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QPushButton" name="botonWebcam">
       <property name="text">
        <string>Abrir ventana de captura de Webcam</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frameChat">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>350</y>
      <width>428</width>
      <height>302</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="botonAbrirChat">
         <property name="text">
          <string>Abrir chat</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="botonCerrarChat">
         <property name="text">
          <string>Cerrar Chat</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Nick cliente:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textoNickCliente">
         <property name="text">
          <string>Cliente</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Nick servidor:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textoNickServidor">
         <property name="text">
          <string>Servidor</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="salidaChatTexto"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLineEdit" name="entradaChatTexto"/>
       </item>
       <item>
        <widget class="QPushButton" name="botonChatEnviar">
         <property name="text">
          <string>Enviar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frameKeylogger">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>200</y>
      <width>231</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="botonActivarKeylogger">
         <property name="text">
          <string>Activar Keylogger</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="botonLimpiarLog">
         <property name="text">
          <string>Limpiar log</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="botonDesactivarKeylogger">
         <property name="text">
          <string>Desactivar Keylogger</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="textTeclas"/>
     </item>
     <item>
      <widget class="QPushButton" name="botonRecibirTeclas">
       <property name="text">
        <string>Recibir teclas</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frameInformacion">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>440</y>
      <width>276</width>
      <height>241</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <widget class="QTextEdit" name="informacionSistemaTexto"/>
     </item>
     <item>
      <widget class="QPushButton" name="botonInformacion">
       <property name="text">
        <string>Actualizar informaci칩n del sistema</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frameSistema">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>153</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_11">
     <item>
      <widget class="QPushButton" name="botonApagar">
       <property name="text">
        <string>Apagar Equipo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="botonReiniciarEquipo">
       <property name="text">
        <string>Reiniciar Equipo</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frameProcesos">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>70</y>
      <width>276</width>
      <height>270</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <widget class="QListWidget" name="listProcesos"/>
     </item>
     <item>
      <widget class="QPushButton" name="botonMatar">
       <property name="text">
        <string>Matar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="botonActualizar">
       <property name="text">
        <string>Actualizar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frameLogin">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>570</y>
      <width>153</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_12">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Direcci칩n XMPP:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textoUsuario"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Contrase침a:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textoContrasena"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="botonConectar">
       <property name="text">
        <string>Conectar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>828</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <widget class="QMenu" name="menuIdioma">
     <property name="title">
      <string>Idioma / Language</string>
     </property>
     <addaction name="actionEspa_ol"/>
     <addaction name="actionEnglish"/>
     <addaction name="actionPortugu_s"/>
    </widget>
    <addaction name="menuIdioma"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuOpciones">
    <property name="title">
     <string>Opciones</string>
    </property>
    <addaction name="actionOpciones"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcerca_de"/>
    <addaction name="actionLicencia"/>
    <addaction name="actionAcerca_de_Qt"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuOpciones"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionOpciones">
   <property name="text">
    <string>Configurar y crear un servidor</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de Demonio</string>
   </property>
  </action>
  <action name="actionLicencia">
   <property name="text">
    <string>Licencia</string>
   </property>
  </action>
  <action name="actionAcerca_de_Qt">
   <property name="text">
    <string>Acerca de Qt</string>
   </property>
  </action>
  <action name="actionConfigurar_puertos">
   <property name="text">
    <string>Configurar puertos</string>
   </property>
  </action>
  <action name="actionEspa_ol">
   <property name="text">
    <string>Espa침ol</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionPortugu_s">
   <property name="text">
    <string>Portugu칡s</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>entradaTexto</tabstop>
  <tabstop>botonEnviar</tabstop>
  <tabstop>salidaTexto</tabstop>
  <tabstop>botonArchivos</tabstop>
  <tabstop>licenciaText</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>botonEnviar</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>shellEnviar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>entradaTexto</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>shellEnviar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botonArchivos</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>abrirVentanaArchivos()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listaOpciones</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>listaOpciones()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>escuchar()</slot>
  <slot>seleccionarServidor()</slot>
  <slot>ping()</slot>
  <slot>cerrarServidor()</slot>
  <slot>listaOpciones()</slot>
  <slot>shellEnviar()</slot>
  <slot>reinciar()</slot>
  <slot>slot1()</slot>
  <slot>abrirVentanaArchivos()</slot>
  <slot>directoriosCambio()</slot>
  <slot>showAboutQt()</slot>
  <slot>licencia()</slot>
  <slot>about()</slot>
 </slots>
</ui>
